src/options.ts

```diff
@@ -33,21 +33,29 @@ export interface ExgesisCompiledOptions {
     allErrors: boolean;
 }
 
-const INT_32_MAX = Math.pow(2, 32) - 1;
- // Actually 18446744073709551616-1, but Javascript doesn't handle integers this large.
-const INT_64_MAX = 18446744073709556000;
-
+const INT_32_MIN = -1 * Math.pow(2, 31);
+const INT_32_MAX = Math.pow(2, 31) - 1;
+// Javascript can only safely support a range of -(2^53 - 1) to (2^53 - 1)
+//      https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER
+const INT_64_MIN = Number.MIN_SAFE_INTEGER;
+const INT_64_MAX = Number.MAX_SAFE_INTEGER;
```

Comment:
I'm not sure if it's better to make this Number.MAX_SAFE_INTEGER or Math.pow(2, 63) - 1.  If someone passes us a JSON object with 4611686018427388001, this is allowed according to the OpenAPI spec, but we are going to round it to 4611686018427388000 which is... odd.

Reply:
I think we should probably allow big numbers, and let it be up to the person using the library to be aware of the fact that JavaScript is weird.  :P